test.js

```diff
@@ -2,8 +2,10 @@
  * Module dependencies.
  */
 
-var assert = require('assert');
-var validate = require('./');
+var assert = require('assert')
+var validate = require('./')
+
+assert.deepStrictEqual = require('is-equal-shallow')
```

Comment:
attaching properties to core libs is generally not a good idea. I'd introduce this as `const equalShallow = require('is-equal-shallow')`.